lib/src/signed_video_h26x_common.c

```diff
@@ -1312,3 +1313,15 @@ signed_video_compare_versions(const char *version1, const char *version2)
 error:
   return status;
 }
+
+bool
+signed_video_is_start_of_stream(uint8_t *sei, size_t sei_size, SignedVideoCodec codec)
```

Comment:
I think it is better to pass in a `signed_video_t` than the `codec` and make this function public in signed_video_auth.h so the validation side can use it.
Also, rename it to `signed_video_is_start_of_stream_sei` and replace sei -> nalu, since it should be possible to pass in any type of NAL Unit.